README.md

```diff
@@ -149,6 +149,7 @@ You must add a service account annotation to the Kubernetes service account so t
 ```bash
 velero install \
     --provider gcp \
+    --plugins velero/velero-plugin-for-gcp:v1.0.0 \
```

Comment:
Let's set the version/tag to `v1.0.0-beta.1` for now. We'll have to update again when we do the GA release.

Reply:
Also need to add this to the previous `velero install` command - L141